app/src/main/java/com/agrawalsuneet/loaders/MainActivity.kt

```diff
@@ -20,18 +20,18 @@ class MainActivity : AppCompatActivity() {
 
         containerLL = findViewById(R.id.container) as LinearLayout
 
-        //initLinearDotsLoader()
-        //initCircularDotsLoader()
-        //initLazyLoader()
-        //initTashieLoader()
-        //initSlidongLoader()
-        //initRotatingCircularDotsLoader()
-
-        //initTrailingCirculerDotsLoader()
-
-        //initZeeLoader()
-
-        //initAllianceLoader()
+        initLinearDotsLoader()
```

Comment:
No need to add these views programmatically as these were for an only testing purpose and for demo code.